+siibra/+internal/initAtlases.m

```diff
@@ -2,9 +2,10 @@
 arguments
     clear_cache (1, 1) logical
 end
-    cached_file_name = '+siibra/cache/atlases.mat';
+    cached_file_name = fullfile('+siibra', 'cache','atlases.mat');
```

Comment:
@SBludau Good catch! Yes, I introduced the usage of fullfile where ever I set a path. This should give us compatibility with windows. Please try again :)